cli/bash/commands/create-one-identity.sh

```diff
@@ -0,0 +1,113 @@
+# Copyright (c) 2019, Arm Limited and affiliates.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#!/bin/bash
+
+set -e
+
+if [ ! -z "$DEBUG" ]; then
+    set -x
+fi
+
+. "$PEP_CLI_DIR/common.sh"
```

Comment:
```suggestion
source "$PEP_CLI_DIR/common.sh"
```

This is only a readability change. Less cryptic to use the command name instead of `.`

Reply:
Quick google search says - "Bash defined source as an alias to the dot. source is a synonym for dot/period '. ' in bash, but not in POSIX sh, so for maximum compatibility use the period." 

Is it safe to say Bash will be available on all platforms?